<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleManager</class>
 <widget class="QWidget" name="ParticleManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particles Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/editor/icons/icons/Particle.png</normaloff>:/editor/icons/icons/Particle.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2">
    <widget class="QLineEdit" name="ln_search"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="table">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>17</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCommandLinkButton" name="btn_Save">
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ParticleViewer" name="viewer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Vitesse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Emission Rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="btn_add">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="btn_delete">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParticleViewer</class>
   <extends>QWidget</extends>
   <header>ParticleManager/ParticleViewer.h</header>
   <container>1</container>
   <slots>
    <slot>setEmitterSpeed(int)</slot>
    <slot>setEmitterRate(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>viewer</receiver>
   <slot>setEmitterSpeed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>905</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>829</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>viewer</receiver>
   <slot>setEmitterRate(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>848</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>883</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>create()</slot>
  <slot>remove()</slot>
  <slot>save()</slot>
  <slot>search(QString)</slot>
 </slots>
</ui>
